examples/petsc/README.md

```diff
@@ -43,11 +43,30 @@ In addition to the common arguments, the following arguments may be set:
 
 ### Command line arguments
 
-The following arguments can be specified for both examples:
+The following arguments can be specified for all of the above examples:
 
 - `-ceed`              - CEED resource specifier
 - `-problem`           - CEED benchmark problem to solve
 - `-degree`            - Polynomial degree of tensor product basis
 - `-qextra`            - Number of extra quadrature points
 - `-test`              - Testing mode (do not print unless error is large)
 - `-benchmark`         - Benchmarking mode (prints benchmark statistics)
+
+### libCEED example to compute surface area using DMPlex - areaplex
+
+This example uses the mass matrix to compute the surface area of a cube, defined via DMPlex. 
+
+To build, run `make area`
+
+To run, `./area -ceed [ceed-resource] -petscspace_degree [degree]`
+
+### Command line arguments
+
+The following arguments can be specified for both examples:
```

Comment:
can be specified for *this example*?

Reply:
Ah great catch, thank you! Forgot to update that line

Reply:
I think you also want examples -> example at the end of the line.

Reply:
Of course! Sorry!